src/db/db.sql

```diff
@@ -159,8 +159,15 @@ CREATE TABLE IF NOT EXISTS access (
 	flags TEXT
 );
 
+-- Holds bans that the bot is tracking (due to sanctions)
+CREATE TABLE bantrack (
+	player INTEGER NOT NULL PRIMARY KEY REFERENCES player(id) DEFERRABLE INITIALLY DEFERRED,
+	expires DATETIME,
+	warning_amount INTEGER
+);
+
 -- Used to hold state between restarts
-CREATE TABLE IF NOT EXISTS pre_restart_state (
+CREATE TABLE pre_restart_state (
 	-- List of players to ping after the bot comes back online
```

Comment:
this script is only run on the initial installation; IF NOT EXISTS is superfluous. Also, this will never delete data in any case, it would just error out if the table exists